{"name": "test_CredenceUrl_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_file_001.Test_Credence_001 object at 0x000001DAC8C4AE50>\n\n    def test_CredenceUrl_002(self):\n        driver = webdriver.Chrome()\n        driver.get(\"https://credence.in/\")\n        if driver.title == \"Credence\":\n            print(\"You are at credence.in\")\n            assert True\n            driver.close()\n        else:\n            print(\"You are at Wrong url\")\n            driver.close()\n>           assert False\nE           assert False\n\ntest_file_001.py:50: AssertionError"}, "start": 1690599159618, "stop": 1690599181685, "uuid": "12ca8067-f629-4ef2-93d0-ff2a0957ac27", "historyId": "b0ec1b20bdc403c83bcb7b36ffc9d4f7", "testCaseId": "b0ec1b20bdc403c83bcb7b36ffc9d4f7", "fullName": "test_file_001.Test_Credence_001#test_CredenceUrl_002", "labels": [{"name": "suite", "value": "test_file_001"}, {"name": "subSuite", "value": "Test_Credence_001"}, {"name": "host", "value": "Tushar-Pc"}, {"name": "thread", "value": "18544-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file_001"}]}